app/services/idv/gpo_mail.rb

```diff
@@ -12,6 +12,15 @@ def mail_spammed?
       max_events? && updated_within_last_month?
     end
 
+    def profile_too_old?
+      return false if !current_user.pending_profile
+
+      max_age_in_days = IdentityConfig.store.gpo_max_profile_age_to_send_letter_in_days
+      min_creation_date = Time.zone.now - max_age_in_days.days
```

Comment:
We could use the `x.days.ago` magic from Rails?

```suggestion
      min_creation_date = IdentityConfig.store.
        gpo_max_profile_age_to_send_letter_in_days.days.ago
```